apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -139,21 +148,14 @@ class SectionProgress extends Component {
               onChange={this.onChangeScript}
             />
           </div>
-          <span style={styles.toggle}>
+          <div style={styles.toggle}>
+            <div style={{...h3Style, ...styles.heading}}>{i18n.viewBy()}</div>
```

Comment:
This is new to me- why do we use the pattern of importing the h3 styles for a div instead of making this an h3 element?

Reply:
I'm not exactly sure why this is like this. Maybe because we use different styling on our H3 elements in react?